### DISCUSSION TITLE
Support for flowtype's Exact objects and Maybe types

### DISCUSSION THREAD

**Discussion by Arooba-git** [babel/babel#15298] [Q&A]
Hi 👋 
I have seen PRs merged for supporting flowtypes' Exact objects. Yet when I'm parsing a project using babel-parser, I'm getting the 'unexpected token' error. Same for _Maybe_ type
<img width="1044" alt="Screen Shot 2022-12-22 at 4 20 05 PM" src="https://user-images.githubusercontent.com/56495631/209079343-aeebcf93-d828-42ef-95d6-16498e4404c6.png">

<img width="1060" alt="Screen Shot 2022-12-22 at 4 19 46 PM" src="https://user-images.githubusercontent.com/56495631/209079395-3d0385ed-14ac-4bcb-a7e1-c34132ec4ad0.png">

Could you guide me to find what's missing? 🤔 

My babel config:
<img width="982" alt="Screen Shot 2022-12-22 at 3 34 53 PM" src="https://user-images.githubusercontent.com/56495631/209079474-71dce92c-e71d-458f-91f1-05dea4d7e4de.png">

**Comment by liuxingbaoyu** [comment]
Can you try removing the ts plugin temporarily?

**Reply by Arooba-git** [reply]
Yeah, these specific flowtype-related errors are gone if I remove the ts preset and plugin. but now, of course, getting a whole bunch of errors for ts syntax 😅
